### ISSUE TITLE
form notice boxes fail color contrast access tests

### ISSUE THREAD

**Issue by GoogleCodeExporter** [davidbelow/blueprintcss#68]
```
What steps will reproduce the problem?
1. Using juicystudio's color contrast analyser extension for firefox, test
http://files.bjorkoy.com/blueprint/tests/forms.html (or test the color
combos on any color contrast analyser 
2. There's not enough difference in color for .error and for .notice, and
there's not enough difference in color nor brightness for success.
3. The colors also need to be changed for the a elements for these classes.

What is the expected output? What do you see instead?
For people with color blindness or reduced vision capabilities,
insufficient contrast in brightness and color can make words unreadable.


What version of the product are you using? On what operating system?
blueprint 0.6, osX, xp2

Please provide any additional information below.

The following color combos do work:
.error      { background: #FBE3E4; color: #8a1f11; border-color: #FBC2C4; }
.notice     { background: #FFF6BF; color: #514721; border-color: #FFD324; }
.success    { background: #E6EFC2; color: #264409; border-color: #C6D880; }
.error a    { color: #8a1f11; }
.notice a   { color: #514721; }
.success a  { color: #264409; }


```

Original issue reported on code.google.com by `jst...@gmail.com` on 14 Jan 2008 at 4:18

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Thanks, I'll change the colors. Let me know if you find any other similar 
mishaps. :)
```

Original comment by `ola...@gmail.com` on 14 Jan 2008 at 6:25
- Changed state: **Fixed**
